lib/v2/mrdx/daily.js

```diff
@@ -0,0 +1,80 @@
+const timezone = require('@/utils/timezone');
+const cheerio = require('cheerio');
+const { parseDate, getElementChildrenInnerText, getHTMLWithCache } = require('./utils');
+const dayjs = require('dayjs');
+const utc = require('dayjs/plugin/utc');
+dayjs.extend(utc);
+
+const got = require('@/utils/got');
+
+module.exports = async (ctx) => {
+    const dateFormatted = dayjs().utcOffset(8).format('YYYYMMDD');
+
+    const baseUrl = `http://mrdx.cn/content/${dateFormatted}`;
+    const link = `${baseUrl}/Page01DK.htm`;
+
+    const indexPageResponse = await got({
+        url: link,
+        method: 'get',
+    });
+    const $ = cheerio.load(indexPageResponse.data);
+
+    // 当日所有文章Url在<a>的`daoxiang` attribute中
+    // 其站点使用**非标准**的HTML语法
+    // eg: `<a class="foo" daoxiang="foo-bar.htm">Lorem ipsum</a>`
+    const passagesList = $('.listdaohang > ul > li a')
+        .toArray()
+        .map((item) => ({
+            link: new URL(`${baseUrl}/${item.attribs.daoxiang}`).href,
+        }));
+
+    const passages = [];
+    const responses = await Promise.all(
+        passagesList.map(async (passage) => {
+            // Return and cache responses processed by cheerio.
+            const response = await getHTMLWithCache(passage.link, ctx);
+            return { ...passage, response };
+        })
+    );
+
+    for (const passage of responses) {
+        const $ = passage.response;
+        // 标题在`h2`,`h3`,`h4`,`h5`中，有多余空格和换行符
+        // 但是Middleware会处理
+        passage.title =
+            getElementChildrenInnerText($('.bggray > .margin15 > h2')[0]) +
+            getElementChildrenInnerText($('.bggray > .margin15 > h3')[0]) +
+            getElementChildrenInnerText($('.bggray > .margin15 > h4')[0]) +
+            getElementChildrenInnerText($('.bggray > .margin15 > h5')[0]);
+
+        // 作者和发布时间
+        // 中间用超多空格分隔
+        // 头尾均有多余空格
+        // eg: 新华每日电讯     2022年12月21日
+        [passage.author, passage.pubDate] = getElementChildrenInnerText($('.shijian')[0])
+            .split(' ')
+            .filter((item) => item !== '')
+            .map((item) => item.trim());
+
+        passage.content = $('.contenttext').html();
+
+        passages.push({
+            title: passage.title,
+            author: passage.author,
+            pubDate: timezone(parseDate(passage.pubDate), 8),
```

Comment:
Consider using RSSHub's utils `parseDate()` instead of reinventing the wheel **with wrong results**
docs: https://docs.rsshub.app/joinus/pub-date.html

![image](https://user-images.githubusercontent.com/11386903/209343974-88344097-c71a-42dc-86df-51b5783b9288.png)